django_stubs_ext/django_stubs_ext/db/models.py

```diff
@@ -0,0 +1,56 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import ClassVar, List, Sequence, Tuple, Union
+
+    from django.db.models import BaseConstraint, Index
+    from typing_extensions import Literal
+
+    from django_stubs_ext import StrOrPromise
+
+    class BaseModelMeta:
```

Comment:
Any thoughts on this naming? BaseModelMeta? ModelMetaBase? TypedModelMeta?

Reply:
Also the module name. I chose `django_stubs_ext.db.models` to match Django's `django.db.models`.

Reply:
I like the way it is: `db.models.BaseModelMeta` 👍